pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -238,7 +223,7 @@ func mlnxNicFromMap(mstData map[string]string, singlePort, firstPort bool) (*mln
 	} else if strings.Contains(linkType, InfinibandLinkType) {
 		fwData.linkType = InfinibandLinkType
 	} else {
-		return nil, fmt.Errorf("mellanox-plugin getMlnxNicFwData(): Unknown link type %s", linkType)
+		glog.Warningf("mellanox-plugin getMlnxNicFwData(): Couldn't read link type %s, preconfigured NIC", linkType)
```

Comment:
Why is it a preconfigured NIC if  linkType is not Infiniband nor Ethernet ?

Reply:
Id expect two different warnings to be emitted in getMlnxNicFwData()

1.  LINK_TYPE_P* attribute was found in the map but it is not one of [ETH, IB]
2. LINK_TYPE_P* attribute was not found in the map

Reply:
fixed